packages/docs/src/locales/en/en.json

```diff
@@ -1134,12 +1134,15 @@
     },
     "VaTimePicker": {
```

Comment:
While scrolling `TimePicker` it may roll back from time to time unpredictably.

![scrolledDownRolledBackSelectedTime](https://user-images.githubusercontent.com/7745899/170985594-ea39b313-b391-4178-bd11-e3a1a43d9750.gif)